components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -235,6 +240,9 @@
 <bnd_topo hgrid="ne240np4" npg="0">atm/cam/topo/USGS-gtopo30_ne240np4_16xconsistentSGH.c20130724.nc</bnd_topo>
 <bnd_topo hgrid="ne512np4"  npg="0">atm/cam/topo/USGS-gtopo30_ne512np4_16xconsistentSGH_20190212.nc</bnd_topo>
 <bnd_topo hgrid="ne1024np4" npg="0">atm/cam/topo/USGS-gtopo30_ne1024np4_16xconsistentSGH_20190528.nc</bnd_topo>
+<bnd_topo hgrid="ne256np4"  npg="2">atm/cam/topo/USGS-gtopo30_ne256np4pg2_16xdel2_20200213.nc</bnd_topo>
```

Comment:
Naming convention is different here...is this the version of the topo before the SGH has been adjusted?

Reply:
I created the ne256np4pg2 topo file from scratch, following workflow 2 on the docs page. For now, ne512 and ne1024 np4pg2 were created using the converter, following workflow 1 on the docs page. The names have different conventions to reflect the different workflows I used.